Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM gcr.io/distroless/static-debian11:debug
```

Comment:
Is there a reason for using debug? Maybe for initial testing and then we remove it?

Reply:
Actually sorry, take that back. I think it will be common to use this as a base image, to embed scripts, or otherwise allow control of options with environment variables. The non-debug image doesn't allow this as it has no shell at all. The only difference is the presence of busybox - what do you think?

Reply:
If that's the case, why don't we use alpine for example? I see the case for distroless for a service, but not for ftw...